internal/operator/operator.go

```diff
@@ -107,7 +109,12 @@ func (o Operator) Start(cfg config.Config) error {
 	probeAddr := viper.GetString("health-probe-bind-address")
 	enableLeaderElection := viper.GetBool("leader-elect")
 	options := ctrl.Options{
-		NewClient:              cluster.ClientBuilderWithOptions(cluster.ClientOptions{CacheUnstructured: true}),
+		NewClient: cluster.ClientBuilderWithOptions(
+			cluster.ClientOptions{
+				CacheUnstructured: true,
+				UncachedObjects:   []client.Object{&eventsv1.Event{}},
```

Comment:
why do we uncache events here?

Reply:
Because there are A LOT of events in the cluster. And this is not the main value chain, but exceptional cases.